### PR TITLE
Added support for eZ Commerce Advanced on Platform.sh

### PR DESCRIPTION
**Pull request by vidarl**
Currently based on branch no_connection_to_solr_2.4, but should be rebased on https://github.com/ezsystems/ezcommerce-advanced/pull/3#pullrequestreview-204773981 once that is merged to 2.4

Edit: Jaroslaw has replaced https://github.com/ezsystems/ezcommerce-advanced/pull/3 with https://github.com/ezsystems/ezcommerce-advanced/pull/7... so needs to rebase on that one, once that one is merged

### CODE REVIEW

**Comment by andrerom** [comment]
@vidarl I guess you can rebase this now once you have time

File: doc/docker/install.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ services:
       for i in $$(seq 1 3); do composer install --no-progress --no-interaction --prefer-dist --no-suggest --optimize-autoloader && s=0 && break || s=$$? && sleep 1; done; (exit $$s);
       mkdir -p web/var;
       php /scripts/wait_for_db.php;
-      php bin/console ezplatform:install ${INSTALL_EZ_INSTALL_TYPE};
+      composer ezcommerce-install
```
Comment (andrerom) [inline doc/docker/install.yml]: This and platform.sh change for install should be changed to use `composer ezplatform-install` upstream, we should not hard code such things in docker and platform.sh config.

**Review by andrerom — APPROVED** [review]
Rebased and pushed this so looks ok'ish now once we fix hard coding of install.

**Comment by andrerom** [comment]
Rebased on top of the installer changes now, so good to go from my side.

**Comment by vidarl** [comment]
Yeah. your rebase looks ok to me too

**Pull request merged**